react/components/LogoutCustomerSession.tsx

```diff
@@ -116,6 +138,10 @@ const LogoutCustomerSession = (props: Props) => {
                 </div>
                 <div className={`${handles.phoneValue} pb5 pl2 c-muted-1`}>{client.phone}</div>
               </div>
+              {customerClass && <div className={`${handles.customerClassContainer} w-100 flex flex-wrap t-small`}>
```

Comment:
What do you think about this change? So we can be sure it's a `true` or `false` value and won't render unwanted things.
```suggestion
              {!!customerClass && <div className={`${handles.customerClassContainer} w-100 flex flex-wrap t-small`}>
```

Ref (and a nice read): https://blog.thoughtspile.tech/2022/01/17/jsx-conditionals/

Reply:
Updated the code & thanks for the article

Reply:
@lariciamota Updated the code & thanks for the article